src/actions/file-search.js

```diff
@@ -173,9 +173,8 @@ export function traverseResults(rev: boolean, editor: Editor) {
 
     if (modifiers) {
       const matchedLocations = matches || [];
-      const results = rev
-        ? findPrev(ctx, query, true, modifiers.toJS())
-        : findNext(ctx, query, true, modifiers.toJS());
+      const findArgs = [ctx, query, true, modifiers.toJS()];
+      const results = rev ? findPrev(...findArgs) : findNext(...findArgs);
```

Comment:
while we're nitting we could also do 

`(rev ? findPrev : findNext)(ctx, query, true, modifiers.toJS())`

not sure which i like more...

Reply:
I thought that was too much :D